src/test/regress/sql/alter_database_propagation.sql

```diff
@@ -56,4 +48,49 @@ alter database regression set lock_timeout from current;
 alter database regression set lock_timeout to DEFAULT;
 alter database regression RESET lock_timeout;
 
+set citus.enable_create_database_propagation=on;
+create database regression2;
+alter database regression2 with CONNECTION LIMIT 100;
+alter database regression2 with IS_TEMPLATE true CONNECTION LIMIT 50;
+alter database regression2 with IS_TEMPLATE false;
+
+
+\set alter_db_tablespace :abs_srcdir '/tmp_check/ts3'
+CREATE TABLESPACE alter_db_tablespace LOCATION :'alter_db_tablespace';
```

Comment:
creating a tablespace with characters that need escaping would also be useful here